into/backends/tests/test_aws.py

```diff
@@ -0,0 +1,253 @@
+import pytest
+import os
+import itertools
+from into import into, resource, S3, discover, CSV, drop, append, SSH
+from into.backends.aws import get_s3_connection
+from into.utils import tmpfile, filetext
+import pandas as pd
+import pandas.util.testing as tm
+import datashape
+
+sa = pytest.importorskip('sqlalchemy')
+boto = pytest.importorskip('boto')
+pytest.importorskip('psycopg2')
```

Comment:
this is specified in the `setup.py` for `redshift_sqlalchemy`
https://github.com/cpcloud/redshift_sqlalchemy/blob/master/setup.py#L21